specification/app/Microsoft.App.DynamicSessions/models/code-execution.tsp

```diff
@@ -0,0 +1,84 @@
+import "./common.tsp";
+using Azure.Core;
+using TypeSpec.Http;
+using TypeSpec.Rest;
+
+namespace Microsoft.App.DynamicSessions;
+
+@doc("The request to execute code.")
+model SessionCodeExecutionRequest {
+  @doc("The code input type of the code execution request.")
+  codeInputType: CodeInputType;
+
+  @doc("The execution type of the code execution request.")
+  executionType: ExecutionType;
+
+  @doc("The string of the code to execute, based on CodeInputType.")
+  code: string;
+
+  @doc("Code execution timeout in seconds.")
+  timeoutInSeconds: int64;
+}
+
+@doc("The result of the code execution.")
+model SessionCodeExecutionResult {
+  @doc("The standard output of the code execution.")
+  stdout?: string;
+
+  @doc("The standard error of the code execution.")
+  stderr?: string;
+
+  #suppress "@azure-tools/typespec-azure-core/no-unknown" ""
+  @doc("The result of the code execution. The type of this field is same as the type of actual result of the code execution after being Json serialized.")
+  executionResult?: unknown;
```

Comment:
If this is a JSON result, please use `Record<unknown>`

Reply:
Thanks, fixed.

Reply:
Hi, had to revert this as this result is not a Json object but a Json value